hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -203,6 +198,43 @@ public void run(Operation op) {
         }
     }
 
+    private void call(Operation op) throws Exception {
+        CallStatus callStatus = op.call();
+
+        switch (callStatus) {
+            case NO_RESPONSE:
+                // todo: currently there is no difference between NO_RESPONSE and OFFLOADED
+                op.afterRun();
+                break;
+            case OFFLOADED:
+                op.afterRun();
+                break;
+            case HAS_RESPONSE:
+                handleResponse(op);
+                afterRun(op);
+                break;
+            case WAIT:
+                nodeEngine.getOperationParker().park((BlockingOperation) op);
```

Comment:
Could we extract it to a method - would be so nice just to see `park(op)`;

Reply:
This one is just a minor thing. Would just look cleaner as a method call, but I'm accepting the PR anyway.